client/ayon_core/hosts/houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pyblish.api
+
+
+class CollectLocalRenderInstances(pyblish.api.InstancePlugin):
+    """Collect instances for local render.
+
+    Agnostic Local Render Collector.
+    """
+
+    # this plugin runs after Collect Render Products
+    order = pyblish.api.CollectorOrder + 0.12
```

Comment:
Why so late? From the code, this can run `pyblish.api.CollectorOrder - 0.49999`

Reply:
I'm re-using collected data `expectedFiles` and they are being collected at `pyblish.api.CollectorOrder + 0.11`

some examples: 

https://github.com/ynput/ayon-core/blob/8f143e446db78783314db8239c5315116bb9c4c5/client/ayon_core/hosts/houdini/plugins/publish/collect_mantra_rop.py#L29

https://github.com/ynput/ayon-core/blob/8f143e446db78783314db8239c5315116bb9c4c5/client/ayon_core/hosts/houdini/plugins/publish/collect_karma_rop.py#L29

https://github.com/ynput/ayon-core/blob/8f143e446db78783314db8239c5315116bb9c4c5/client/ayon_core/hosts/houdini/plugins/publish/collect_redshift_rop.py#L29

Reply:
But those are not creating new instances.

Reply:
yes, but I'm reusing their collected data to create my new instances.

Reply:
It's hard because there is some dependency in the collectors. 

- `CollectHoudiniReviewData - CollectorOrder + 0.13` needs to run after `CollectRopFrameRange - CollectorOrder` and `CollectLocalRenderInstances - CollectorOrder + 0.12`
- `CollectLocalRenderInstances - CollectorOrder + 0.12` needs to run after Collect Render Products e.g. `CollectMantraROPRenderProducts - CollectorOrder + 0.11`
- `CollectMantraROPRenderProducts - CollectorOrder + 0.11` needs to run after `CollectFrames - CollectorOrder + 0.1`
- `CollectFrames - CollectorOrder + 0.1` needs to run after `CollectRopFrameRange  - CollectorOrder`

Mostly, I used `0.01` step.